packages/components/src/internal/components/ontology/OntologyTreeSearchContainer.tsx

```diff
@@ -1,25 +1,81 @@
-import React, { ChangeEvent, FC, memo, MouseEvent, useCallback, useEffect, useMemo, useState } from 'react';
+import React, { ChangeEvent, FC, memo, MouseEvent, ReactNode, useCallback, useEffect, useMemo, useState } from 'react';
 
 import { Alert, searchUsingIndex } from '../../..';
 
 import { ConceptModel, OntologyModel, PathModel } from './models';
-import { fetchAlternatePaths } from './actions';
+import { fetchAlternatePaths, getOntologyDetails } from './actions';
 
 const CONCEPT_CATEGORY = 'concept';
 const SEARCH_LIMIT = 20;
 
+interface OntologySearchInputProps {
+    ontologyId: string;
+    initCode?: string; //initial concept code
+    inputName: string;
+    className: string;
+    searchPathClickHandler: (code: string) => void;
+    showBrowserHandler?: () => void;
+    fieldName?: string;
+}
+
+export const OntologySearchInput: FC<OntologySearchInputProps> = memo(props => {
```

Comment:
I would declare `OntologySearchInput` below `OntologyTreeSearchContainer` since it uses the tree container. I'm guessing IntelliJ is warning about this.

Reply:
moved